features/campaign_can_only_be_created_by_artist.feature

```diff
@@ -0,0 +1,28 @@
+@javascript
+Feature: A Campaign can only be created by an Artist
+    As a System owner, 
+    In order to eliminate fake and non-serious Campaigns
+    We would like the Campaigns to only be created by Artists.
+
+    Background:
+        Given the following users exist
+        | email             | role      |
+        | user@fan.se       | fan       |
+        | user@artist.se    | artist    |
+
+    Scenario: Fan tries to access Create Campaign page via link
+        Given I am logged in as 'user@fan.se'
+        And I try to access 'Create Campaign' page
```

Comment:
try to use all of Given When Then
`When I try to access 'Create Campaign' page`

Reply:
I understand what you mean, but I think the current wording makes more sense in this case.